themes/edx.org/lms/templates/dashboard.html

```diff
@@ -141,29 +142,39 @@
 <section class="dashboard" id="dashboard-main">
 <main class="main-container" id="main" aria-label="Content" tabindex="-1">
   <%
-    from openedx.core.djangoapps.demographics.api.status import show_user_demographics
+    from openedx.core.djangoapps.demographics.api.status import (
+        show_user_demographics, show_call_to_action_for_user
+    )
     from openedx.core.djangoapps.enrollments.api import serialize_enrollments
     serialized_course_enrollments = serialize_enrollments(course_enrollments)
     demographics_enabled = getattr(settings, 'ENABLE_DEMOGRAPHICS_COLLECTION', False)
-    display_demographics_banner = demographics_enabled and show_user_demographics(
-      user, enrollments=serialized_course_enrollments, entitlements=course_entitlements
+    display_demographics_banner = (
+        demographics_enabled and
+        show_user_demographics(user, enrollments=serialized_course_enrollments, entitlements=course_entitlements) and
+        show_call_to_action_for_user(user)
     )
     account_mfe_url = getattr(settings, 'ACCOUNT_MICROFRONTEND_URL', '') or ''
   %>
   % if display_demographics_banner:
-    <a class="btn" href="${account_mfe_url}#demographics-information">
-      <div
-        class="demographics-banner d-flex justify-content-left align-items-center flex-column flex-lg-row py-1 px-4 mb-2 mb-lg-4">
+  <a id="demographics-banner-link" class="btn" href="${account_mfe_url}#demographics-information">
```

Comment:
Wrapping a standard interactive thing with another interactive thing is kinda non-standard. It makes the wrapping link a confusing mouthfull of text for screen reader users. I understand the desire to have the whole banner be a touch target. In the future I suggest attaching a click handler to the div instead of adding the wrapping link.

Reply:
Thank you for the feedback on this. We will keep it in mind for future implementations.